src/HttpTracer/HttpTracer.csproj

```diff
@@ -1,44 +1,36 @@
-    <PackageTags>xamarin, windows, ios, android, http, tracing</PackageTags>
-    <Title>HttpTracer - A Http tracking lib</Title>
-    <Description>A simple http tracing library to write request and response information to your output window. Making your life easier when debugging http calls!</Description>
-    <Copyright>2017 &amp; Toronto Mobile .NET Developers Group</Copyright>
-    <RepositoryUrl>https://github.com/TorontoMobileDevelopers/HttpTracer</RepositoryUrl>
-    <PackageReleaseNotes>See: https://github.com/TorontoMobileDevelopers/HttpTracer</PackageReleaseNotes>
-
-    <LangVersion>default</LangVersion>
-
-    <DefineConstants>$(DefineConstants);</DefineConstants>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <RootNamespace>HttpTracer</RootNamespace>
-    <GenerateDocumentationFile Condition=" '$(Configuration)' == 'Release' ">true</GenerateDocumentationFile>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Condition=" '$(TargetFramework)' == 'uap10.0.16299' " Include="Microsoft.NETCore.UniversalWindowsPlatform " Version="6.0.6" />
-    <PackageReference Include="MSBuild.Sdk.Extras" Version="1.2.0" PrivateAssets="All" />
-
-    <Reference Condition=" '$(TargetFramework)' == 'Xamarin.Mac20' " Include="System.Net.Http" />
-  </ItemGroup>
-
-
-  <Import Project="$(MSBuildSDKExtrasTargets)" Condition="Exists('$(MSBuildSDKExtrasTargets)')" />
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;netstandard1.4</TargetFrameworks>
+        <Product>$(AssemblyName) ($(TargetFramework))</Product>
+        <AssemblyVersion>1.0.1.0</AssemblyVersion>
+        <AssemblyFileVersion>1.0.1.0</AssemblyFileVersion>
+        <Version>1.0.1.0</Version>
+        <PackageVersion>1.0.1.0</PackageVersion>
+        <Authors>Dylan Berry; Daniel Causer; Chase Florell</Authors>
+        <PackageId>HttpTracer</PackageId>
+
+        <NeutralLanguage>en</NeutralLanguage>
+        <PackageLicenseUrl>https://github.com/BSiLabs/HttpTracer/blob/master/LICENSE</PackageLicenseUrl>
+        <Owners>Toronto Mobile .NET Developers Group</Owners>
+        <PackageProjectUrl>https://github.com/BSiLabs/HttpTracer</PackageProjectUrl>
+        <PackageIconUrl>https://raw.githubusercontent.com/BSiLabs/HttpTracer/master/art/HttpTracer.png</PackageIconUrl>
+        <Summary>A simple http tracing library to write request and response information to your output window. Making your life easier when debugging http calls!</Summary>
+        <PackageTags>xamarin, windows, ios, android, http, tracing</PackageTags>
+        <Title>HttpTracer - A Http tracking lib</Title>
+        <Description>A simple http tracing library to write request and response information to your output window. Making your life easier when debugging http calls!</Description>
+        <Copyright>2019 &amp; Toronto Mobile .NET Developers Group</Copyright>
```

Comment:
I thought the copyright year should be the year that the product was first released.

Reply:
It is supposed to roll forward every year if you continue to work on it